src/Microsoft.Data.SqlClient/src/Microsoft/Data/SqlClient/Server/ValueUtilsSmi.cs

```diff
@@ -3174,7 +3174,7 @@ private static SqlMoney GetSqlMoney_Unchecked(SmiEventSink_Default sink, ITypedG
 
             long temp = getters.GetInt64(sink, ordinal);
             sink.ProcessMessagesAndThrow();
-#if NETCOREAPP && NET7_0_OR_GREATER
+#if NET7_0_OR_GREATER
```

Comment:
Wonder if NET7_0_OR_GREATER could be replaced with NET8_0_OR_GREATER now?

Reply:
@roji Maybe you have some input?

Reply:
Yep, makes sense to me.